extendeddatasquare.go

```diff
@@ -260,6 +260,52 @@ func (eds *ExtendedDataSquare) Width() uint {
 	return eds.width
 }
 
+// FlattenedEDS returns the extended data square as a flattened slice of bytes.
+func (eds *ExtendedDataSquare) FlattenedEDS() [][]byte {
+	return eds.Flattened()
+}
+
+// FlattenedODS returns the original data square as a flattened slice of bytes.
+func (eds *ExtendedDataSquare) FlattenedODS() (flattened [][]byte) {
+	flattened = make([][]byte, eds.originalDataWidth*eds.originalDataWidth)
+	for i := uint(0); i < eds.originalDataWidth; i++ {
+		row := eds.Row(i)
+		for j := uint(0); j < eds.originalDataWidth; j++ {
+			flattened[(i*eds.originalDataWidth)+j] = row[j]
+		}
+	}
+	return flattened
+}
+
+// Equals returns true if other is equal to eds.
+func (eds *ExtendedDataSquare) Equals(other *ExtendedDataSquare) bool {
```

Comment:
What does Equals mean here? Don't all the shares have to be the same for this to be true? i.e. `eds.squareRow[i] == other.squareRow[i]`